nim/tests/test_routes.py

```diff
@@ -0,0 +1,149 @@
+import json
+
+import pytest
+
+from nim.game import (
+    DEFAULT_MIN,
+    DEFAULT_MAX,
+    DEFAULT_PILES
+)
+
+from nim.app import app
```

Comment:
The rule isn't to have an empty line between imports.
Here it should look like this:
```
import json

import pytest

from nim.app import app
from nim.game import (
    DEFAULT_MIN,
    DEFAULT_MAX,
    DEFAULT_PILES
)
```
Please refer to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) for more information.